### ISSUE TITLE
A Error

### ISSUE THREAD

**Issue by Moosbee** [SapuSeven/BetterUntis#252]
<details>
<summary>Logs</summary>

```
Anfragen-ID: 4
Fehlercode: 3
Fehlermeldung: o7.b: Field 'staffInfo' is required for type with serial name 'com.sapuseven.untis.models.untis.timetable.PeriodText', but it was missing
```
</details>

**Additional information**

- Android version: _10_
- BetterUntis version: _3.4.7 (31)_
- Installation source: _com.android.vending_

**Comment by lm41** [comment]
This was fixed in version 3.4.8

**Comment by SapuSeven** [comment]
As mentioned, this has been fixed in version 3.4.8. You can opt-in as a beta tester on Google Play to receive the update sooner.
